Timefy/View/HomeVC.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  HomeViewController.swift
+//  Timefy
+//
+//  Created by Doğancan Mavideniz on 10.09.2022.
+//
+
+import UIKit
+
+class HomeVC: UIViewController, UITableViewDataSource, UITableViewDelegate {
+
+    @IBOutlet private weak var tableView: UITableView!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.view.backgroundColor = .appWhite
+        self.tableView.dataSource = self
+        self.tableView.delegate = self
+        self.registerTableViewCells()
+    }
+
+    private func registerTableViewCells() {
+        let textFieldCell = UINib(nibName: "CustomTableViewCell",
+            bundle: nil)
+        self.tableView.register(textFieldCell,
+            forCellReuseIdentifier: "CustomTableViewCell")
```

Comment:
indentation sıkıntısı var. Bu satırı seçip control + I yaparsan aşağıdaki gibi olur 
```suggestion
        let textFieldCell = UINib(nibName: "CustomTableViewCell",
                                                 bundle: nil)
        self.tableView.register(textFieldCell,
                                               forCellReuseIdentifier: "CustomTableViewCell")
```